### PR TITLE
Fix/dgj 11 branding

### PR DESCRIPTION
**Pull request by iman-jamali-fw**
## Summary

Fixing the style issues with branding ticket #11 

## Changes
fixing include
- header
- navigation items, sign out button on small screen
- footer links
- buttons style fixed

More details in this [doc](https://docs.google.com/document/d/1Hjbg-NDL3WRvXPXw8PqDPV9eEvSLO_EEasi0-r6mfrQ/edit)

### CODE REVIEW

File: forms-flow-web/src/components/Form/List.scss [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+.flex-item-right {
+  justify-content: flex-end;
+  display: flex;
+  flex-direction: row;
+  flex: 50%;
+}
+.btn-left{
+    margin: 10px;
+    height: 50px;
```
Comment (iman-jamali-fw) [inline forms-flow-web/src/components/Form/List.scss:87]: only this `height: 50px;` property changed to fix the issue with button style on this page

**Review by simensma-fresh — APPROVED** [review]
🎨

**Review by matbusby-fw — APPROVED** [review]
Nice work!  It's odd reviewing some of this code.  I noticed a few things that were from the original formsflow code (not written by you) that might merit a note, but I don't think we want to go down the rabbit hole of patching code we're not writing. (example of this was not using a key on the mapped list items)

**Comment by iman-jamali-fw** [comment]
> Nice work! It's odd reviewing some of this code. I noticed a few things that were from the original formsflow code (not written by you) that might merit a note, but I don't think we want to go down the rabbit hole of patching code we're not writing. (example of this was not using a key on the mapped list items)

Agreed! certainly style of the app is a little confusing mixing form.io style with form-flow and our own :)

**Pull request merged**